### PR TITLE
Add Missing Concept Exercise Scaffolds + create `Options` concept + minor fixes to scripts

### PR DESCRIPTION
**Pull request by 0xNeshi**
Towards #103 
Towards #255 

Most of the "Files changed" are just newly added blank files.

This PR should include all of the concept exercise scaffolds and placeholders.
- For exercises that I copied over from other tracks, I added placeholder files and appropriate fields.
- For exercises I created specifically for this track, I either implemented the metadata ([cryptographer](https://github.com/exercism/cairo/pull/272/files#diff-a44417ae740fcbbd8a89d1987db4aae4129842eea26144d1bb1039c4f81d166b)) or implemented just the exercise implementation+stub+tests ([chrono-realms-time-tree](https://github.com/exercism/cairo/pull/272/files#diff-597c0b057d3bf2208136849269ffc9b25f4eddb3e27a4d9beef1692faa13fe83), [chrono-realms](https://github.com/exercism/cairo/pull/272/files#diff-cacbd427b8a02b91dcafb6009dd78f46d82c271a1bcd1fd340cd02a4dd54e08e), [the-realm-of-echoes](https://github.com/exercism/cairo/pull/272/files#diff-5cba43f20ea12cf8277df61a3f90383ab3331069fb26ddf2b96b73f38b5a62c6)) and I added my username as part of the `authors` array (alongside whoever implements the missing content). **<-- These are the exercises where the most of the stuff is going on.**

### CODE REVIEW

**Comment by BNAndras** [comment]
This PR has a lot going on so it'll take me a few days to review.

**Comment by 0xNeshi** [comment]
@BNAndras I always hated huge PRs. Luckily, out of the 160 files of the PR, only around 20-30 are worth reviewing, others are stubs (see updated description).
Still, even in those 20-30 files there's some content to go through. In retrospect, maybe should've broken this down to multiple PRs, each introducing a single concept placeholder.

**Comment by 0xNeshi** [comment]
@BNAndras I'm not sure how far along you are with reviewing this PR, but if you haven't had the time (or the will) to review this, I would separate it out into many smaller PRs

**Comment by BNAndras** [comment]
I’ll have some time this week to take a look at that subset of files. At the moment, I can review it as a single large PR.

File: exercises/concept/chrono-realms/.docs/instructions.md
Comment (BNAndras) [inline exercises/concept/chrono-realms/.docs/instructions.md:1]: One sentence per line for the Exercism style guide

File: exercises/concept/chrono-realms/.docs/introduction.md
Comment (BNAndras) [inline exercises/concept/chrono-realms/.docs/introduction.md:1]: This file is required and introduces the concepts used within this exercise. Once the concepts are introduced, the instructions.md content provide the exercise-specific information (i.e. what the tasks are).

File: exercises/concept/chrono-realms/.meta/config.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "authors": [
+    "<your_gh_username>",
```
Comment (BNAndras) [inline exercises/concept/chrono-realms/.meta/config.json:3]:
```suggestion
```

Exercise is already authored. Any subsequent updates by someone not an author would make them a contributor

File: exercises/concept/chrono-realms-time-tree/.meta/config.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "authors": [
+    "<your_gh_username>",
```
Comment (BNAndras) [inline exercises/concept/chrono-realms-time-tree/.meta/config.json:3]:
```suggestion
```

File: exercises/concept/airport-robot/.meta/config.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      ".meta/exemplar.cairo"
+    ],
+    "invalidator": [
+      "Scarb.toml"
+    ]
+  },
+  "forked_from": [
+    "go/airport-robot"
+  ],
+  "blurb": "<blurb>"
```
Comment (BNAndras) [inline exercises/concept/airport-robot/.meta/config.json:22]: If this is forked from Go, we can surely borrow its blurb.

File: exercises/concept/chrono-realms-time-tree/.meta/config.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+      "tests/chrono_realms_time_tree.cairo"
+    ],
+    "exemplar": [
+      ".meta/exemplar.cairo"
+    ],
+    "invalidator": [
+      "Scarb.toml"
+    ]
+  },
+  "blurb": "<blurb>"
```
Comment (BNAndras) [inline exercises/concept/chrono-realms-time-tree/.meta/config.json:20]: The blurb is required

File: exercises/concept/chrono-realms/.meta/config.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+      "tests/chrono_realms.cairo"
+    ],
+    "exemplar": [
+      ".meta/exemplar.cairo"
+    ],
+    "invalidator": [
+      "Scarb.toml"
+    ]
+  },
+  "blurb": "<blurb>"
```
Comment (BNAndras) [inline exercises/concept/chrono-realms/.meta/config.json:20]: Missing blurb

File: exercises/concept/cryptographer/.docs/introduction.md
Comment (BNAndras) [inline exercises/concept/cryptographer/.docs/introduction.md:1]: One sentence per line per style guide

File: exercises/concept/cryptographer/.docs/instructions.md
Comment (BNAndras) [inline exercises/concept/cryptographer/.docs/instructions.md:1]: One sentence per line

File: exercises/concept/cryptographer/.docs/hints.md
Comment (BNAndras) [inline exercises/concept/cryptographer/.docs/hints.md:1]: One sentence per line.

Also, it's unclear whether this hint is for a particular task or the exercise as a whole.

File: exercises/concept/cryptographer/.meta/config.json
Comment (BNAndras) [inline exercises/concept/cryptographer/.meta/config.json:1]: Several exercise files are empty for this slug so I can't say I'd consider this exercise authored. I'd recommend either adding the missing content including the exemplar or drop yourself as an author for now.

File: exercises/concept/election-day/.meta/config.json [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+      ".meta/exemplar.cairo"
+    ],
+    "invalidator": [
+      "Scarb.toml"
+    ]
+  },
+  "forked_from": [
+    "go/election-day"
+  ],
+  "blurb": "<blurb>"
```
Comment (BNAndras) [inline exercises/concept/election-day/.meta/config.json:22]: We can probably borrow the blurb from Go.

File: exercises/concept/gross-store/.meta/config.json [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+      ".meta/exemplar.cairo"
+    ],
+    "invalidator": [
+      "Scarb.toml"
+    ]
+  },
+  "forked_from": [
+    "go/gross-store"
+  ],
+  "blurb": "<blurb>"
```
Comment (BNAndras) [inline exercises/concept/gross-store/.meta/config.json:22]: Borrow the blurb from Go?

File: exercises/concept/health-statistics/.docs/hints.md
Comment (BNAndras) [inline exercises/concept/health-statistics/.docs/hints.md:1]: One sentence per line

File: exercises/concept/health-statistics/.docs/instructions.md
Comment (BNAndras) [inline exercises/concept/health-statistics/.docs/instructions.md:1]: This file reads like the introduction. The instructions file should outline what tasks need to be done for the exercise to be completed. Also, one sentence per line please.

File: exercises/concept/health-statistics/.docs/introduction.md
Comment (BNAndras) [inline exercises/concept/health-statistics/.docs/introduction.md:1]: One sentence per line.

File: exercises/concept/health-statistics/.docs/introduction.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+```rust
+impl ItemImpl of ImplTrait {
+    // initializes and returns a new instance of our Item struct
+    fn new() -> Item {
+        Item {}
+    }
+}
+```
+
+With that brief introduction to the syntax of structs and traits out of the way, go ahead and take a look at the [instructions](instructions.md) for this exercise!
```
Comment (BNAndras) [inline exercises/concept/health-statistics/.docs/introduction.md:47]:
```suggestion
```

When viewed on Exercism, the instructions will be appended to the end of the introduction so this isn't needed.

File: exercises/concept/health-statistics/Scarb.toml
Comment (BNAndras) [inline exercises/concept/health-statistics/Scarb.toml:1]: This file isn't needed if we don't have a test suite or exemplar to work with yet. Other Scarb.toml files have been empty.

File: exercises/concept/low-power-embedded-game/.meta/design.md [hunk 9]
```diff
@@ -21,6 +21,7 @@ Introduce the student to tuples and how to work with them.
 
 ## Resources to refer to
 
+- [Cairo Book - The Tuple Type](https://book.cairo-lang.org/ch02-02-data-types.html#the-tuple-type)
+
 ### Hints
```
Comment (BNAndras) [inline exercises/concept/low-power-embedded-game/.meta/design.md:26]: This file isn't exposed to any students or mentors so we don't need a hints section.

File: exercises/concept/magical-measurements/.docs/instructions.md
Comment (BNAndras) [inline exercises/concept/magical-measurements/.docs/instructions.md:1]: One sentence per line

File: exercises/concept/magician-in-training/.meta/config.json [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+      ".meta/exemplar.cairo"
+    ],
+    "invalidator": [
+      "Scarb.toml"
+    ]
+  },
+  "forked_from": [
+    "gleam/magician-in-training"
+  ],
+  "blurb": "<blurb>"
```
Comment (BNAndras) [inline exercises/concept/magician-in-training/.meta/config.json:22]: Can probably use Gleam's blurb here

File: exercises/concept/red-vs-blue-darwin-style/.meta/config.json [hunk 11]
```diff
@@ -0,0 +1,23 @@
...
+      ".meta/exemplar.cairo"
+    ],
+    "invalidator": [
+      "Scarb.toml"
+    ]
+  },
+  "forked_from": [
+    "csharp/red-vs-blue-darwin-style"
+  ],
+  "blurb": "<blurb>"
```
Comment (BNAndras) [inline exercises/concept/red-vs-blue-darwin-style/.meta/config.json:22]: Can we use C#'s blurb here?

File: exercises/concept/role-playing-game/.meta/config.json [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+      ".meta/exemplar.cairo"
+    ],
+    "invalidator": [
+      "Scarb.toml"
+    ]
+  },
+  "forked_from": [
+    "rust/role-playing-game"
+  ],
+  "blurb": "<blurb>"
```
Comment (BNAndras) [inline exercises/concept/role-playing-game/.meta/config.json:22]: Use Rust's blurb here

File: exercises/concept/the-farm/.meta/config.json [hunk 13]
```diff
@@ -0,0 +1,23 @@
...
+    ],
+    "invalidator": [
+      "Scarb.toml"
+    ]
+  },
+  "forked_from": [
+    "go/the-farm"
+  ],
+  "blurb": "<blurb>"
+}
```
Comment (BNAndras) [inline exercises/concept/the-farm/.meta/config.json:23]: Use Go's blurb here?

File: exercises/concept/the-realm-of-echoes/.docs/instructions.md
Comment (BNAndras) [inline exercises/concept/the-realm-of-echoes/.docs/instructions.md:1]: One sentence per line

File: exercises/concept/the-realm-of-echoes/.meta/config.json [hunk 14]
```diff
@@ -0,0 +1,21 @@
...
+      "tests/the_realm_of_echoes.cairo"
+    ],
+    "exemplar": [
+      ".meta/exemplar.cairo"
+    ],
+    "invalidator": [
+      "Scarb.toml"
+    ]
+  },
+  "blurb": "<blurb>"
```
Comment (BNAndras) [inline exercises/concept/the-realm-of-echoes/.meta/config.json:20]: Blurb missing

File: exercises/concept/the-realm-of-echoes/.meta/config.json [hunk 15]
```diff
@@ -0,0 +1,21 @@
+{
+  "authors": [
+    "0xNeshi",
+    "<your_gh_username>"
```
Comment (BNAndras) [inline exercises/concept/the-realm-of-echoes/.meta/config.json:4]:
```suggestion
```

Exercise looks authored so there's no need for a placeholder here.

File: exercises/concept/welcome-to-tech-palace/.meta/config.json [hunk 16]
```diff
@@ -0,0 +1,23 @@
...
+      ".meta/exemplar.cairo"
+    ],
+    "invalidator": [
+      "Scarb.toml"
+    ]
+  },
+  "forked_from": [
+    "go/welcome-to-tech-palace"
+  ],
+  "blurb": "<blurb>"
```
Comment (BNAndras) [inline exercises/concept/welcome-to-tech-palace/.meta/config.json:22]: Can you use Go's blurb?

File: exercises/concept/chrono-realms/.docs/instructions.md
Comment (0xNeshi) [reply exercises/concept/chrono-realms/.docs/instructions.md:1]: Given that many files do not currently follow this rule, I would like to address this as part of a separate issue, see https://github.com/exercism/cairo/issues/285

File: exercises/concept/chrono-realms/.meta/config.json (hunk 1 above)
Comment (0xNeshi) [reply exercises/concept/chrono-realms/.meta/config.json:3]: I think anyone implementing the rest of this exercise could be considered a co-author, see https://github.com/exercism/cairo/pull/272#discussion_r1777017124

File: exercises/concept/airport-robot/.meta/config.json (hunk 3 above)
Comment (0xNeshi) [reply exercises/concept/airport-robot/.meta/config.json:23]: Will be implemented as part of the relevant ticket/PR, see https://github.com/exercism/cairo/pull/272#discussion_r1777017124

File: exercises/concept/chrono-realms-time-tree/.meta/config.json (hunk 4 above)
Comment (0xNeshi) [reply exercises/concept/chrono-realms-time-tree/.meta/config.json:20]: See https://github.com/exercism/cairo/pull/272#discussion_r1777017124

File: exercises/concept/health-statistics/.docs/instructions.md
Comment (0xNeshi) [reply exercises/concept/health-statistics/.docs/instructions.md:1]: There's not much else to say except "you have this struct, implement the stub methods in the impl block".
Do you think the rest of the content (everything below "Your goal is to implement the stubbed out methods on the `User` `struct` defined in the `impl` block.") should be removed?

File: exercises/concept/cryptographer/.docs/hints.md
Comment (0xNeshi) [reply exercises/concept/cryptographer/.docs/hints.md:1]: It's for the whole exercise. What do you think would be a better wording?

**Review by BNAndras — APPROVED** [review]

**Pull request merged**